libraries/Microsoft.Bot.Connector/Authentication/IOAuthConfiguration.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Bot.Connector.Authentication
+{
+    /// <summary>
+    /// Configuration for OAuth client credential authentication.
+    /// </summary>
+    public interface IOAuthConfiguration
+    {
```

Comment:
The `OAuthConfiguration` class is a value object and usually you don't need to supply an interface for a value object because, by definition, it doesn't have any behavior that would ever need to be substituted. Consider removing this to simplify.

Reply:
If you do this, make sure to rename the file to match. 👍

Reply:
The interface was there because I was planning to have another implementation so didn't want to couple on the concrete type, but I don't think I'll need it. If that proves the case, I'll certainly remove the interface

Reply:
Done, I won't need the interface as there won't be different implementations